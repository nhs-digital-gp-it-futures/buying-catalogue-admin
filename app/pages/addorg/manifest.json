{
  "title": "Find an organisation",
  "description": "Enter the correct Organisation Data Service (ODS) code to find the organisation you're looking for.",
  "isMultiQuestion": true,
  "questions": [
    {
      "id": "odsCode",
      "mainAdvice": "Enter ODS code"
    }
  ],
  "continueButtonText": "Continue"
}
