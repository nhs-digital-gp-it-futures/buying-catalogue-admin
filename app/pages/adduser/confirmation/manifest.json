{
  "backLinkText": "Back to dashboard",
  "backLinkHref": "/organisations/{{organisationId}}",
  "title": "{{userAdded}} account added",
  "description": "The user has been sent an email so they can set their password."
}
