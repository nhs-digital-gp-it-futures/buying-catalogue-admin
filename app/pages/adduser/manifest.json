{
  "title": "Add new user",
  "description": "Provide the following information to make a new user account.",
  "orgNameSubheading": "Organisation name",
  "isMultiQuestion": true,
  "questions": [
    {
      "id": "firstName",
      "mainAdvice": "First name"
    },
    {
      "id": "lastName",
      "mainAdvice": "Last name"
    },
    {
      "id": "phoneNumber",
      "mainAdvice": "Telephone number"
    },
    {
      "id": "emailAddress",
      "mainAdvice": "Email address"
    }
  ],
  "addUserButtonText": "Add user"
}
