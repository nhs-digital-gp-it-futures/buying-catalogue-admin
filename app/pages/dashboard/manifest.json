{
  "title": "Manage organisations and users",
  "description": "Edit existing account information or add new user to an organisation.",
  "columnInfo": [
    {
      "data": "Organisation name"
    },
    {
      "data": "ODS code"
    }
  ],
  "columnClass": "nhsuk-grid-column-one-half"
}
