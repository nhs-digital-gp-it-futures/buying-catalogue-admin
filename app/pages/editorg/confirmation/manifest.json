{
  "backLinkText": "Back to dashboard",
  "title": "{{organisationName}} updated",
  "description": "This organisation's details have been updated.",
  "dataTestId": "edit-organisation-confirmation"
}
