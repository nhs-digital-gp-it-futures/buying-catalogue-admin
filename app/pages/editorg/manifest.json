{
  "description": "Change the Organisation End User Agreement status for this organisation.",
  "odsCodeHeading": "ODS code",
  "organisationNameHeading": "Organisation name",
  "primaryRoleIdHeading": "Primary Role ID",
  "addressHeading": "Business address",
  "catalogueAgreementCheckboxText": "Organisation End User Agreement has been signed",
  "saveButtonText": "Save"
}
