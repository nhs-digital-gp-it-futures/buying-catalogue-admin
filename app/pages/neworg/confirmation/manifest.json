{
  "backLinkText": "Back to dashboard",
  "title": "{{orgName}} created",
  "description": "This organisation has been created on the Buying Catalogue.",
  "dataTestId": "create-org-confirmation"
}
