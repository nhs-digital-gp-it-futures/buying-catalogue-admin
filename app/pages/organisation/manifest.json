{
  "description": "These are the details for the organisation and its users.",
  "columnInfo": [
    {
      "data": "Name",
      "link": true
    },
    {
      "data": "Telephone number"
    },
    {
      "data": "Email address"
    },
    {}
  ],
  "columnClass": "nhsuk-grid-column-one-quarter"
}
