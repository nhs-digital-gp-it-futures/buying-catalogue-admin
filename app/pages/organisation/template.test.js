import { createTestHarness } from '../../test-utils/testHarness';
import manifest from './manifest.json';

const setup = {
  template: {
    path: 'pages/organisation/template.njk',
  },
};
const mockData = {
  organisationId: 'org1',
  orgName: 'Greater Manchester CCG',
  odsCode: 'X01',
  primaryRoleId: 'ID123',
  address: ['12 Station Road', 'Leeds', 'West Yorkshire', 'LS15 5FG'],
  isCatalogueAgreementSigned: true,
  agreementSigned: true,
  addUserButtonHref: '/organisations/org1/adduser',
  users: [
    [{
      data: 'John Smith',
    }, {
      data: '07777777777',
    }, {
      data: 'john.smith@email.com',
    }, {
      tag: false,
    }],
    [{
      userId: 'user2',
    }, {
      data: 'Daisy Chain',
    }, {
      data: '07777777778',
    }, {
      data: 'daisy.chain@email.com',
    }, {
      tag: {
        dataTestId: 'account-disabled-tag-user2',
        classes: 'bc-c-tag-outline nhsuk-u-font-size-16',
        text: 'ACCOUNT DISABLED',
      },
    }],
  ],
};

const mockContext = {
  ...manifest,
  ...mockData,
};

describe('organisations dashboard page', () => {
  it('should render a backLink to the dashboard page', createTestHarness(setup, (harness) => {
    harness.request(mockContext, ($) => {
      const homepageBackLink = $('[data-test-id="go-back-link"]');
      expect(homepageBackLink.length).toEqual(1);
      expect(homepageBackLink.text().trim()).toEqual('Back');
      expect($(homepageBackLink).find('a').attr('href')).toEqual('/organisations');
    });
  }));

  it('should render a title', createTestHarness(setup, (harness) => {
    harness.request(mockContext, ($) => {
      const title = $('h1[data-test-id="org-page-title"]');
      expect(title.length).toEqual(1);
      expect(title.text().trim()).toEqual(mockContext.orgName);
    });
  }));

  it('should render a description', createTestHarness(setup, (harness) => {
    harness.request(mockContext, ($) => {
      const description = $('[data-test-id="org-page-description"]');
      expect(description.length).toEqual(1);
      expect(description.text().trim()).toEqual(mockContext.description);
    });
  }));

  it('should render a organisation details subheading', createTestHarness(setup, (harness) => {
    harness.request(mockContext, ($) => {
      const subheading = $('h3[data-test-id="organisation-details-subheading"]');
      expect(subheading.length).toEqual(1);
      expect(subheading.text().trim()).toEqual(mockContext.orgSubheading);
    });
  }));

  it('should render a edit organisation button', createTestHarness(setup, (harness) => {
    harness.request(mockContext, ($) => {
      const addOrgButton = $('[data-test-id="edit-org-button"] a');
      expect(addOrgButton.length).toEqual(1);
      expect(addOrgButton.text().trim()).toEqual(mockContext.editOrgButtonText);
      expect(addOrgButton.attr('href')).toEqual('#');
    });
  }));

  it('should render ods code', createTestHarness(setup, (harness) => {
    harness.request(mockContext, ($) => {
      const odsCodeHeading = $('[data-test-id="org-page-ods-code-heading"]');
      const odsCode = $('[data-test-id="org-page-ods-code"]');
      expect(odsCodeHeading.length).toEqual(1);
      expect(odsCodeHeading.text().trim()).toEqual(mockContext.odsCodeHeading);
      expect(odsCode.length).toEqual(1);
      expect(odsCode.text().trim()).toEqual(mockContext.odsCode);
    });
  }));

  it('should render primary role id', createTestHarness(setup, (harness) => {
    harness.request(mockContext, ($) => {
      const primaryRoleIdHeading = $('[data-test-id="org-page-primary-role-id-heading"]');
      const primaryRoleId = $('[data-test-id="org-page-primary-role-id"]');
      expect(primaryRoleIdHeading.length).toEqual(1);
      expect(primaryRoleIdHeading.text().trim()).toEqual(mockContext.primaryRoleIdHeading);
      expect(primaryRoleId.length).toEqual(1);
      expect(primaryRoleId.text().trim()).toEqual(mockContext.primaryRoleId);
    });
  }));

  it('should render address', createTestHarness(setup, (harness) => {
    harness.request(mockContext, ($) => {
      const addressHeading = $('[data-test-id="org-page-address-heading"]');
      const address1 = $('[data-test-id="org-page-address-1"]');
      const address2 = $('[data-test-id="org-page-address-2"]');
      const address3 = $('[data-test-id="org-page-address-3"]');
      const address4 = $('[data-test-id="org-page-address-4"]');
      expect(addressHeading.length).toEqual(1);
      expect(addressHeading.text().trim()).toEqual(mockContext.addressHeading);
      expect(address1.length).toEqual(1);
      expect(address1.text().trim()).toEqual(mockContext.address[0]);
      expect(address2.length).toEqual(1);
      expect(address2.text().trim()).toEqual(mockContext.address[1]);
      expect(address3.length).toEqual(1);
      expect(address3.text().trim()).toEqual(mockContext.address[2]);
      expect(address4.length).toEqual(1);
      expect(address4.text().trim()).toEqual(mockContext.address[3]);
    });
  }));

  it('should render a agreement signed checked statement', createTestHarness(setup, (harness) => {
    harness.request(mockContext, ($) => {
      const agreementSigned = $('[data-test-id="agreement-signed-checked-statement"]');
      expect(agreementSigned.length).toEqual(1);
      expect(agreementSigned.text().trim()).toEqual(mockContext.agreementSignedText);
    });
  }));

  it('should render an accounts subheading with org name', createTestHarness(setup, (harness) => {
    harness.request(mockContext, ($) => {
      const subheading = $('h3[data-test-id="accounts-subheading"]');
      expect(subheading.length).toEqual(1);
      expect(subheading.text().trim()).toEqual(`${mockContext.accountsSubheading} ${mockContext.orgName}`);
    });
  }));

  it('should render an add user button', createTestHarness(setup, (harness) => {
    harness.request(mockContext, ($) => {
      const addOrgButton = $('[data-test-id="add-user-button"] a');
      expect(addOrgButton.length).toEqual(1);
      expect(addOrgButton.text().trim()).toEqual(mockContext.addUserButtonText);
      expect(addOrgButton.attr('href')).toEqual(`/organisations/${mockContext.organisationId}/adduser`);
    });
  }));

  it('should render the table component', createTestHarness(setup, (harness) => {
    harness.request(mockContext, ($) => {
      const orgTable = $('[data-test-id="user-table"]');
      expect(orgTable.length).toEqual(1);
      expect(orgTable.find('[data-test-id="table"]').length).toEqual(1);
      expect(orgTable.find('[data-test-id="table-headings"]').length).toEqual(1);
      expect(orgTable.find('[data-test-id="column-heading"]').length).toEqual(mockContext.columnInfo.length);
    });
  }));
});
