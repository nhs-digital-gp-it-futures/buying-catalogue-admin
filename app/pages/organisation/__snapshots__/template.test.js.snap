// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`organisations dashboard page the page should render 1`] = `
"<main class=\\"nhsuk-main-wrapper\\" id=\\"maincontent\\" data-test-id=\\"main-content\\">
        
  <div class=\\"nhsuk-width-container\\" data-test-id=\\"organisation-page\\">
    <div data-test-id=\\"go-back-link\\">
      <div class=\\"nhsuk-back-link\\">
  <a class=\\"nhsuk-back-link__link\\" href=\\"/organisations\\">
  <svg class=\\"nhsuk-icon nhsuk-icon__chevron-left\\" xmlns=\\"http://www.w3.org/2000/svg\\" viewBox=\\"0 0 24 24\\" aria-hidden=\\"true\\">
    <path d=\\"M8.5 12c0-.3.1-.5.3-.7l5-5c.4-.4 1-.4 1.4 0s.4 1 0 1.4L10.9 12l4.3 4.3c.4.4.4 1 0 1.4s-1 .4-1.4 0l-5-5c-.2-.2-.3-.4-.3-.7z\\"></path>
  </svg>
  Go back</a>
</div>

    </div>
  
    <div class=\\"nhsuk-grid-row\\">
      <div class=\\"nhsuk-grid-column-two-thirds\\">
        <h1 data-test-id=\\"org-page-title\\" class=\\"nhsuk-u-margin-top-7 nhsuk-u-margin-bottom-3\\">Greater Manchester CCG</h1>
        <p data-test-id=\\"org-page-description\\" class=\\"nhsuk-heading-l nhsuk-u-font-size-22\\">These are the details for the organisation and its users.</p>
        <h2 data-test-id=\\"organisation-details-subheading\\">Organisation details</h2>
      </div>
    </div>

    <dl class=\\"nhsuk-summary-list nhsuk-u-padding-margin-5\\">
      <div class=\\"nhsuk-summary-list__row\\">
        <dt class=\\"nhsuk-summary-list__key\\">
          ODS code
        </dt>
        <dd class=\\"nhsuk-summary-list__value\\">
          X01
        </dd>
      </div>
      <div class=\\"nhsuk-summary-list__row\\">
        <dt class=\\"nhsuk-summary-list__key\\">
          Primary Role ID
        </dt>
        <dd class=\\"nhsuk-summary-list__value\\">
          ID123
        </dd>
      </div>
      <div class=\\"nhsuk-summary-list__row\\">
        <dt class=\\"nhsuk-summary-list__key\\">
          Business address
        </dt>
        <dd class=\\"nhsuk-summary-list__value\\">
          
            <div data-test-id=\\"org-page-address-1\\">12 Station Road</div>
          
            <div data-test-id=\\"org-page-address-2\\">Leeds</div>
          
            <div data-test-id=\\"org-page-address-3\\">West Yorkshire</div>
          
            <div data-test-id=\\"org-page-address-4\\">LS15 5FG</div>
          
        </dd>
      </div>
      <div class=\\"nhsuk-summary-list__row\\">
        <dt class=\\"nhsuk-summary-list__key\\">
          Contract
        </dt>
        <dd class=\\"nhsuk-summary-list__value\\">
          <a data-test-id=\\"edit-org-button\\" href=\\"/organisations/org1/edit\\" class=\\"float-right nhsuk-u-margin-left-2\\">Change</a>
          <div data-test-id=\\"agreement-signed-statement\\">
            
              Organisation End User Agreement has been signed
            
          </div>
        </dd>
      </div>
    </dl>
  
    <div class=\\"nhsuk-grid-row\\">
      <div class=\\"nhsuk-grid-column-two-thirds\\">
        <h2>User accounts</h2>
        <p>These are the user accounts for Greater Manchester CCG.</p>
      </div>
    </div>

    <div data-test-id=\\"add-user-button\\">
      <div class=\\"nhsuk-action-link\\">
  <a class=\\"nhsuk-action-link__link\\" href=\\"/organisations/org1/adduser\\">
    <svg class=\\"nhsuk-icon nhsuk-icon__arrow-right-circle\\" xmlns=\\"http://www.w3.org/2000/svg\\" viewBox=\\"0 0 24 24\\" aria-hidden=\\"true\\">
      <path d=\\"M0 0h24v24H0z\\" fill=\\"none\\"></path>
      <path d=\\"M12 2a10 10 0 0 0-9.95 9h11.64L9.74 7.05a1 1 0 0 1 1.41-1.41l5.66 5.65a1 1 0 0 1 0 1.42l-5.66 5.65a1 1 0 0 1-1.41 0 1 1 0 0 1 0-1.41L13.69 13H2.05A10 10 0 1 0 12 2z\\"></path>
    </svg>
    <span class=\\"nhsuk-action-link__text\\">Add new user</span>
  </a>
</div>

    </div>

    <div data-test-id=\\"users-table\\" class=\\"nhsuk-u-font-size-19 even-table\\">
      
  <table data-test-id=\\"table\\" class=\\"\\" style=\\"page-break-inside:auto\\">
    
    
      <thead data-test-id=\\"table-headings\\" style=\\"display:table-header-group;\\">
        
          <tr><th data-test-id=\\"column-heading-0\\" class=\\"nhsuk-u-padding-bottom-0 \\" style=\\"width:\\">
            <div data-test-id=\\"column-heading-0-data\\" class=\\"nhsuk-u-font-weight-bold\\">Name</div>
            
            
          </th>
        
          <th data-test-id=\\"column-heading-1\\" class=\\"nhsuk-u-padding-bottom-0 \\" style=\\"width:\\">
            <div data-test-id=\\"column-heading-1-data\\" class=\\"nhsuk-u-font-weight-bold\\">Telephone number</div>
            
            
          </th>
        
          <th data-test-id=\\"column-heading-2\\" class=\\"nhsuk-u-padding-bottom-0 \\" style=\\"width:\\">
            <div data-test-id=\\"column-heading-2-data\\" class=\\"nhsuk-u-font-weight-bold\\">Email address</div>
            
            
          </th>
        
          <th data-test-id=\\"column-heading-3\\" class=\\"nhsuk-u-padding-bottom-0 \\" style=\\"width:\\">
            <div data-test-id=\\"column-heading-3-data\\" class=\\"nhsuk-u-font-weight-bold\\"></div>
            
            
          </th>
        
          <th data-test-id=\\"column-heading-4\\" class=\\"nhsuk-u-padding-bottom-0 \\" style=\\"width:\\">
            <div data-test-id=\\"column-heading-4-data\\" class=\\"nhsuk-u-font-weight-bold\\"></div>
            
            
          </th>
        
      </tr></thead>
    

    <tbody>
    
      <tr data-test-id=\\"table-row-0\\" style=\\"page-break-inside:avoid; page-break-after:auto\\">
        
          
          <td class=\\"nhsuk-u-padding-bottom-0\\" style=\\"border-bottom-style: solid\\">

            
              <a data-test-id=\\"user-name-user1\\" href=\\"/organisations/org1/user1\\">John Smith</a>
            

            
          </td>
        
          
          <td class=\\"nhsuk-u-padding-bottom-0\\" style=\\"border-bottom-style: solid\\">

            
              
  
    
    <div data-test-id=\\"user-phone-user1\\" class=\\"bc-c-data-text \\">07777777777</div>
  

            

            
          </td>
        
          
          <td class=\\"nhsuk-u-padding-bottom-0\\" style=\\"border-bottom-style: solid\\">

            
              
  
    
    <div data-test-id=\\"user-email-user1\\" class=\\"bc-c-data-text \\">john.smith@email.com</div>
  

            

            
          </td>
        
          
          <td class=\\"nhsuk-u-padding-bottom-0\\" style=\\"border-bottom-style: solid\\">

            
              
  

            

            
          </td>
        
      </tr>
    
      <tr data-test-id=\\"table-row-1\\" style=\\"page-break-inside:avoid; page-break-after:auto\\">
        
          
          <td class=\\"nhsuk-u-padding-bottom-0\\" style=\\"border-bottom-style: solid\\">

            
              <a data-test-id=\\"user-name-user2\\" href=\\"/organisations/org1/user2\\">Daisy Chain</a>
            

            
          </td>
        
          
          <td class=\\"nhsuk-u-padding-bottom-0\\" style=\\"border-bottom-style: solid\\">

            
              
  
    
    <div data-test-id=\\"user-phone-user2\\" class=\\"bc-c-data-text \\">07777777778</div>
  

            

            
          </td>
        
          
          <td class=\\"nhsuk-u-padding-bottom-0\\" style=\\"border-bottom-style: solid\\">

            
              
  
    
    <div data-test-id=\\"user-email-user2\\" class=\\"bc-c-data-text \\">daisy.chain@email.com</div>
  

            

            
          </td>
        
          
          <td class=\\"nhsuk-u-padding-bottom-0\\" style=\\"border-bottom-style: solid\\">

            
              <div>
                
  <div data-test-id=\\"account-disabled-tag-user2\\" class=\\"bc-c-tag nhsuk-u-font-weight-bold bc-c-tag-outline nhsuk-u-font-size-14\\">ACCOUNT DISABLED</div>

              </div>
            

            
          </td>
        
      </tr>
    
    </tbody>
    
  </table>

    </div>

    <div class=\\"nhsuk-grid-row\\">
      <div class=\\"nhsuk-grid-column-two-thirds\\">
        <h2>Related organisations</h2>
        <p>These are the organisations Greater Manchester CCG can create orders for.</p>
      </div>
    </div>

    <div data-test-id=\\"add-organisation-button\\">
      <div class=\\"nhsuk-action-link\\">
  <a class=\\"nhsuk-action-link__link\\" href=\\"/admin/organisations/proxy/org1\\">
    <svg class=\\"nhsuk-icon nhsuk-icon__arrow-right-circle\\" xmlns=\\"http://www.w3.org/2000/svg\\" viewBox=\\"0 0 24 24\\" aria-hidden=\\"true\\">
      <path d=\\"M0 0h24v24H0z\\" fill=\\"none\\"></path>
      <path d=\\"M12 2a10 10 0 0 0-9.95 9h11.64L9.74 7.05a1 1 0 0 1 1.41-1.41l5.66 5.65a1 1 0 0 1 0 1.42l-5.66 5.65a1 1 0 0 1-1.41 0 1 1 0 0 1 0-1.41L13.69 13H2.05A10 10 0 1 0 12 2z\\"></path>
    </svg>
    <span class=\\"nhsuk-action-link__text\\">Add an organisation</span>
  </a>
</div>

    </div>

    <div data-test-id=\\"related-org-table\\" id=\\"related-org-table\\" class=\\"nhsuk-u-font-size-19 even-table\\">
      
  <table data-test-id=\\"table\\" class=\\"\\" style=\\"page-break-inside:auto\\">
    
    
      <thead data-test-id=\\"table-headings\\" style=\\"display:table-header-group;\\">
        
          <tr><th data-test-id=\\"column-heading-0\\" class=\\"nhsuk-u-padding-bottom-0 \\" style=\\"width:\\">
            <div data-test-id=\\"column-heading-0-data\\" class=\\"nhsuk-u-font-weight-bold\\">Organisation name</div>
            
            
          </th>
        
          <th data-test-id=\\"column-heading-1\\" class=\\"nhsuk-u-padding-bottom-0 \\" style=\\"width:\\">
            <div data-test-id=\\"column-heading-1-data\\" class=\\"nhsuk-u-font-weight-bold\\">ODS code</div>
            
            
          </th>
        
          <th data-test-id=\\"column-heading-2\\" class=\\"nhsuk-u-padding-bottom-0 \\" style=\\"width:\\">
            <div data-test-id=\\"column-heading-2-data\\" class=\\"nhsuk-u-font-weight-bold\\"></div>
            
            
          </th>
        
      </tr></thead>
    

    <tbody>
    
      <tr data-test-id=\\"table-row-0\\" style=\\"page-break-inside:avoid; page-break-after:auto\\">
        
          
          <td class=\\"nhsuk-u-padding-bottom-0\\" style=\\"border-bottom-style: solid\\">

            
              
  
    
    <div data-test-id=\\"related-org-name-348a04d0-1f3e-49d5-b652-2e9db4c2d423\\" class=\\"bc-c-data-text \\">Yorkshire and Humber – H&amp;J Commissioning Hub</div>
  

            

            
          </td>
        
          
          <td class=\\"nhsuk-u-padding-bottom-0\\" style=\\"border-bottom-style: solid\\">

            
              
  
    
    <div data-test-id=\\"related-org-odsCode-348a04d0-1f3e-49d5-b652-2e9db4c2d423\\" class=\\"bc-c-data-text \\">14N</div>
  

            

            
          </td>
        
          
          <td class=\\"nhsuk-u-padding-bottom-0\\" style=\\"border-bottom-style: solid\\">

            
              <a data-test-id=\\"related-org-remove-348a04d0-1f3e-49d5-b652-2e9db4c2d423\\" href=\\"/admin/organisations/removeproxy/b7ee5261-43e7-4589-907b-5eef5e98c085/348a04d0-1f3e-49d5-b652-2e9db4c2d423\\">Remove</a>
            

            
          </td>
        
      </tr>
    
      <tr data-test-id=\\"table-row-1\\" style=\\"page-break-inside:avoid; page-break-after:auto\\">
        
          
          <td class=\\"nhsuk-u-padding-bottom-0\\" style=\\"border-bottom-style: solid\\">

            
              
  
    
    <div data-test-id=\\"related-org-name-2459261a-d8f0-464a-bff7-45b2d7adb1ef\\" class=\\"bc-c-data-text \\">Yorkshire and Humber Commissioning Hub</div>
  

            

            
          </td>
        
          
          <td class=\\"nhsuk-u-padding-bottom-0\\" style=\\"border-bottom-style: solid\\">

            
              
  
    
    <div data-test-id=\\"related-org-odsCode-2459261a-d8f0-464a-bff7-45b2d7adb1ef\\" class=\\"bc-c-data-text \\">13V</div>
  

            

            
          </td>
        
          
          <td class=\\"nhsuk-u-padding-bottom-0\\" style=\\"border-bottom-style: solid\\">

            
              <a data-test-id=\\"related-org-remove-2459261a-d8f0-464a-bff7-45b2d7adb1ef\\" href=\\"/admin/organisations/removeproxy/b7ee5261-43e7-4589-907b-5eef5e98c085/2459261a-d8f0-464a-bff7-45b2d7adb1ef\\">Remove</a>
            

            
          </td>
        
      </tr>
    
    </tbody>
    
  </table>

    </div>
  </div>

      </main>"
`;
